Content.Server/GameObjects/Components/PlaceableSurfaceComponent.cs

```diff
@@ -12,38 +13,74 @@ namespace Content.Server.GameObjects.Components
     [ComponentReference(typeof(SharedPlaceableSurfaceComponent))]
     public class PlaceableSurfaceComponent : SharedPlaceableSurfaceComponent, IInteractUsing
     {
-        private bool _isPlaceable;
+        private bool isPlaceable;
+        private bool placeCentered;
+        private Vector2 positionOffset;
 
         [ViewVariables(VVAccess.ReadWrite)]
         public override bool IsPlaceable
         {
-            get => _isPlaceable;
+            get => isPlaceable;
             set
             {
-                if (_isPlaceable == value)
+                if (isPlaceable == value)
                 {
                     return;
                 }
 
-                _isPlaceable = value;
+                isPlaceable = value;
 
                 Dirty();
             }
         }
 
+        [ViewVariables(VVAccess.ReadWrite)]
+        public override bool PlaceCentered
+        {
+            get => placeCentered;
+            set
+            {
+                if (placeCentered == value)
+                {
+                    return;
+                }
+
+                placeCentered = value;
+
+            }
+        }
+
+        [ViewVariables(VVAccess.ReadWrite)]
+        public override Vector2 PositionOffset
+        {
+            get => positionOffset;
+            set
+            {
+                if (positionOffset.EqualsApprox(value))
+                {
+                    return;
+                }
+
+                positionOffset = value;
+
+            }
+        }
+
         [ViewVariables]
         int IInteractUsing.Priority => -10;
 
         public override void ExposeData(ObjectSerializer serializer)
         {
             base.ExposeData(serializer);
 
-            serializer.DataField(ref _isPlaceable, "IsPlaceable", true);
+            serializer.DataField(ref isPlaceable, "IsPlaceable", true);
+            serializer.DataField(ref placeCentered, "PlaceCentered", false);
+            serializer.DataField(ref positionOffset, "PositionOffset", Vector2.Zero);
```

Comment:
I'm not gonna force you to rename the legacy/old YAML field, but these new ones should be camelCase at least.

Reply:
I've renamed the new YAML fields to use camelCase